commit 88d11d9028b031c36f8932434717c0a17ab5b759 (refs/changes/68/2068/2)
Author: Jared Morrow <jm@joyent.com>
Date:   2017-07-21T08:20:00-06:00 (2 years, 3 months ago)
    
    MANTA-2900 mlocate should not need to setupMantaBuckets
    Reviewed by: David Pacheco <dap@joyent.com>

diff --git a/bin/mlocate b/bin/mlocate
index d592789..993e949 100755
--- a/bin/mlocate
+++ b/bin/mlocate
@@ -81,6 +81,7 @@ function createMorayClient(opts, cb) {
 
     var log = LOG.child({component: 'moray'}, true);
     opts.log = log;
+    opts.readOnly = true;
 
     var client = new libmanta.createMorayClient(opts);
 
